iroha/src/sumeragi.rs

```diff
@@ -381,45 +384,47 @@ impl Sumeragi {
                     .expect("Failed to get System Time.")
             && tx_receipt.is_valid(&self.network_topology)
         {
-            self.transactions_awaiting_receipts
+            let transaction_was_waiting = self
```

Comment:
unnecessary local variable, if you made it to make condition more verbose and clear, it's better to encapsulate it somewhere.